src/PageHeader/features.stories.tsx

```diff
@@ -65,6 +65,24 @@ export const WithLeadingAndTrailingVisuals = () => (
   </Box>
 )
 
+export const WithComponentAsATitle = () => (
+  <Box sx={{padding: 3}}>
+    <PageHeader>
+      <PageHeader.TitleArea>
+        <Breadcrumbs>
+          <Breadcrumbs.Item href="#">...</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="#">primer</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="#">react</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="#">src</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="#">PageHeader</Breadcrumbs.Item>
+          <Breadcrumbs.Item href="#">PageHeader.tsx</Breadcrumbs.Item>
+        </Breadcrumbs>
+        <h2 className="visually-hidden">Visually Hidden Title</h2>
```

Comment:
I noticed that in the [preview for this example](https://primer-5c67b4ddbf-13348165.drafts.github.io/storybook/?path=/story/drafts-components-pageheader-features--with-component-as-a-title), the heading isn't actually hidden. Is this expected?

Reply:
😬 No it is NOT EXPECTED 😅 I'll fix it